connectors/vmware/src/centreon/vmware/logger.pm

```diff
@@ -50,10 +50,18 @@ use warnings;
 use Sys::Syslog qw(:standard :macros);
 use IO::Handle;
 
-my %severities = (1 => LOG_INFO,
-                  2 => LOG_ERR,
-                  4 => LOG_DEBUG);
-
+my %severities = (
+    1 => LOG_INFO,
+    2 => LOG_ERR,
+    4 => LOG_DEBUG,
+    7 => LOG_CRIT
+);
+my %human_severities = (
```

Comment:
Why the 1 is error in human_severities and LOG_INFO in severities ?

Reply:
I know it's not your code but can you change or delete the comment line 72 ? it seem wrong from what I understand.